tests/test_time_frequency.py

```diff
@@ -377,5 +377,15 @@ def _test(layer, input_batch, allclose_func, atol=1e-4):
     )
 
 
+@pytest.mark.parametrize('hop_ratio', [0.5, 0.25, 0.125])
+def test_frequency_aware_conv2d(hop_ratio):
+    input_shape = (4, 28, 28, 3)
+    x = tf.random.normal(input_shape)
+    y = FreqAwareConv(
+     10, 3, activation='relu', input_shape=input_shape[1:])(x)
+    print(y.shape)
+    np.testing.assert_equal(y.shape, (4, 26, 26, 10))
+
```

Comment:
once we have a separate method for creating the frequency map, let's add it in the test!

Reply:
great! I'll work on this next

Reply:
thanks for your work again!

Reply:
I've used ```test_frequency_aware_conv2d_channel_add```, let me know if it's ok

Reply:
instead of comparing the frequency map at t=0 and t=1, can't we compare the whole map vs `np.tile(np.arrange(), (time, 1))`? as long as that is correct, i can finalize the other stuff.

Reply:
yeah I think that's correct